CONTRIBUTING.md

```diff
@@ -131,7 +136,7 @@ Special front matter fields we use:
 In order to link to other docs pages, please use this format only:
 
 ```markdown
-... see the [gsctl reference](/reference/gsctl/) for details. ...
+... see the [gsctl reference]({{< relref "/ui-api/gsctl" >}}) for details. ...
```

Comment:
This page is not rendered by hugo, right? The `relref` shortcode does not work on GitHub

Reply:
Right. Must have been a mass edit failure. 👍

Reply:
Wait, that's only a `relref` example for contributors. The code is not supposed to work.

Reply:
ah yes. the ```markdown line :)